packages/contracts-core/Makefile

```diff
@@ -0,0 +1,32 @@
+include ../../make/solidity.Makefile
+
+devnet-clean: ## Delete all devnet data and docker containers
+	rm -rf deployments/chain_a/*.json
+	rm -rf deployments/chain_b/*.json
+	rm -rf deployments/chain_c/*.json
+
+	# TODO: this should also delete broadcast/**/42, broadcast/**/43, broadcast/**/44
+	rm -rf broadcast/42/*
+	rm -rf broadcast/43/*
+	rm -rf broadcast/44/*
+
+	cd ../../docker/devnet && docker-compose down --volumes
+
+devnet-up: ## This should be run to start the devnet docker-containers. This does not deploy anything.
+	cd ../../docker/devnet && docker-compose up -d --build
+
+devnet-deploy: ## This should be run exactly once to deploy the contracts to the devnet.
+	# backup the .env in case there's real data in there
+	cp .env /tmp/.env.bak || true
+	cp devnet.env .env
+
```

Comment:
Deploy Factory here prior to deploying everything?

Reply:
Yeah, this is definitely an option